README.md

```diff
@@ -1,14 +1,96 @@
-## VOMS configuration files for OSG
+## VOMS Configuration Files for OSG
 
-VOMS configuration files used to create vo-package / vo-client RPMS (https://twiki.grid.iu.edu/bin/view/SoftwareTeam/CreateVOClient)
+### Files Included
 
-### To rebuild vomsdir from vomses, run
+-   `vomses`
+    This file contains information about VOMS servers, one per line.
+
```

Comment:
Minor nitpick: Splitting up these two sentences doesn't look great when rendered in the README

```suggestion
```